azure/spoke/variables.tf

```diff
@@ -33,14 +33,19 @@ variable "address_space" {
 
   validation {
     error_message = "Must be valid IPv4 CIDR."
```

Comment:
`Must be valid IPv4 CIDR.`

It's technically a list of valid CIDR's. Perhaps better phrasing would be:

`All Address Spaces must be Valid IPv4 CIDR`